libV2/server/API/metrics/ingestMetric.js

```diff
@@ -1,11 +1,15 @@
 const errors = require('../../../errors');
 const { UtapiMetric } = require('../../../models');
 const { client: cacheClient } = require('../../../cache');
+const { convertTimestamp } = require('../../../utils');
 
 async function ingestMetric(ctx, params) {
     let metrics;
     try {
-        metrics = params.body.map(m => new UtapiMetric(m));
+        metrics = params.body.map(m => new UtapiMetric({
```

Comment:
What about having the UtapiMetric constructor do this itself?

Reply:
I don't think it's a good idea to have the constructor of a model change the passed data.

Reply:
Oh it's not a basic class its the custom JOI model thing I reviewed a while back... then yea ignore me thats not a good idea.